src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -483,13 +483,25 @@ public String getServicePhraseName(Interface service) {
   }
 
   /**
+   * The type name of the Grpc server class. This needs to match what Grpc generates for the
+   * particular language.
+   */
+  public String getGrpcServerTypeName(Interface service) {
```

Comment:
This is mostly copied from `getGrpcClientTypeName` and changed "Client" to "Server". I think this is just as general. If we want to turn it into `getNotImplementedString`, we should probably change both.